.NET/Microsoft.Recognizers.Text.DataDrivenTests/Number/TestNumberRecognizer_English.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Recognizers.Text.Number;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.DataDrivenTests.Tests
+{
+    [TestClass]
+    public class TestNumberRecognizer_English
+    {
+        [TestMethod]
+        public void RecognizeNumber()
+        {
+            var actual = NumberRecognizer.RecognizeNumber("192", Culture.English);
```

Comment:
The purpose of these new tests is to make sure the new Recognize*() methods work, right? But could we also make them data-driven (i.e. read json specs like the others)?
I imagine we'll have more tests in the future to cover other cases, like culture fallback for example, or "en" vs "en-US" cases; and it would be good to cover all supported cultures, not just test the English impl.

Reply:
Not a major issue though. Feel free to ignore it if the effort is too high for the benefit now.

Reply:
Yes, actually they helped us designing the Recognize*() methods and serve as usage examples. We agree with you that we might want to convert them into data-driven tests.
We will keep the unit tests updates in our radar and we'll probably create a new branch and PR.
It would be great to discuss the best approach for testing the SDK including all the latest changes.

Reply:
Thanks. A separate PR would be great.
When you say "SDK" you mean more than just the Recognizers-Text project, correct? We can start a thread about that.

Reply:
👍